restcomm/restcomm.commons/src/main/java/org/restcomm/connect/commons/common/http/CustomHttpClientBuilder.java

```diff
@@ -58,12 +68,21 @@ private CustomHttpClientBuilder() {
     }
 
     private static CloseableHttpClient defaultClient = null;
+    private static CloseableHttpAsyncClient closeableHttpAsyncClient = null;
```

Comment:
not sure putting sync and async factory in same class is meaningful (maybe a separated async factory would have work better). Lets keep it like this by now...